src/site/stages/build/process-cms-exports/transformers/helpers.js

```diff
@@ -4,6 +4,45 @@ const { sortBy, unescape, pick, omit } = require('lodash');
 const moment = require('moment-timezone');
 const { readEntity } = require('../helpers');
 
+const mediaImageStyles = [
```

Comment:
@swirtSJW Can you verify that all these names + dimensions are accurate?

Reply:
@swirtSJW made a [comment here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/13215#issuecomment-690681276) about it